lib/money_tracker.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'transaction.rb'
+require 'date'
+
+class MoneyTracker
+
+  attr_reader :balance, :transaction
+
+  def initialize(balance = 0)
+    @balance = balance
```

Comment:
What are the downsides of keeping a running total of the balance, rather than calculating it when it's needed?